src/ansys/acp/core/_tree_objects/model.py

```diff
@@ -255,29 +267,29 @@ def _create_stub(self) -> model_pb2_grpc.ObjectServiceStub:
     )
 
     @classmethod
-    def from_file(cls, *, path: _PATH, channel: Channel) -> Model:
+    def _from_file(cls, *, path: _PATH, server_wrapper: ServerWrapper) -> Model:
```

Comment:
Why was this method renamed to `_from_file` where `from_fe_file` is unchanged.

Reply:
Good point, I think `from_fe_file` should also be renamed. I think the methods accepting a `ServerWrapper` should be internal, since we don't expect users to call them directly and we may want to change how the server is represented in the future.